aten/src/ATen/native/cuda/Sort.cpp

```diff
@@ -60,37 +63,9 @@ std::tuple<Tensor &,Tensor &> sort_out_stable_cuda(const Tensor & self, c10::opt
   TORCH_CHECK(self_dtype != ScalarType::ComplexFloat && self_dtype != ScalarType::ComplexDouble,
```

Comment:
welp, more cuda-only checks. There probably isn't much we can do about these though... so I think it's fine to leave them here.